### ISSUE TITLE
possible issue: webapp bombarding fcd.

### ISSUE THREAD

**Issue by thinkingparticle** [terraswap/terraswap-web-app#56]
probably you have already seen this. 
I wanted to swap mir for luna, was slow, checked console, it sends 120 query_msg to fcd. Thought maybe something is wrong? not sure.

![image](https://user-images.githubusercontent.com/17741554/142651899-1eaa31d3-ea3c-4e85-97a7-6a992d5a1b61.png)

**Comment by JoowonYun** [comment]
Yes by checking the number of all cases for routing. We are refactoring a way to reduce this.
